cmd/aws.go

```diff
@@ -130,3 +130,10 @@ func setupProviderCacher(cfg aws.Config) (*providercacher.CachingQueuePoller, er
 
 	return providercacher.NewCachingQueuePoller(cachingQueue, providerCacher)
 }
+
+func setupIPNIPublisherStore(cfg aws.Config) *store.AdStore {
+	ipniStore := aws.NewS3Store(cfg.Config, cfg.IPNIStoreBucket, cfg.IPNIStorePrefix)
+	chunkLinksTable := aws.NewDynamoProviderContextTable(cfg.Config, cfg.ChunkLinksTableName)
+	metadataTable := aws.NewDynamoProviderContextTable(cfg.Config, cfg.MetadataTableName)
+	return store.NewPublisherStore(ipniStore, chunkLinksTable, metadataTable, store.WithMetadataContext(metadata.MetadataContext))
+}
```

Comment:
nit: since this is exactly the same initialization that is done in `setupIPNIPipeline` (this file, lines 104-107), I'd call `setupIPNIPublisherStore` a bit earlier and pass the resulting store as a new parameter to `setupIPNIPipeline`.